chpl/chpl-resources/src/main/resources-dev/jobs.xml

```diff
@@ -307,7 +310,7 @@
                     </entry>
                     <entry>
                         <key>acb</key>
-                        <value>Drummond Group☺ICSA Labs☺UL LLC☺SLI Compliance</value>
+                        <value>3☺6☺1☺4</value>
```

Comment:
What do you think about making these comma-delimited? I think we used the smiley character because it was something we could reliably split on (unlikely to be used in an ACB name) but moving to IDs frees us up to use a more normal delimiter.